test/memcache_protocol_test.cpp

```diff
@@ -337,14 +340,14 @@ BOOST_AUTO_TEST_CASE ( cas_gets_test ) {
     std::string retrieved;
     boost::uint64_t cas_value = 0u;
     BOOST_CHECK_NO_THROW ( mc << memcache::set("key", test) );
-    BOOST_CHECK_THROW    ( mc << memcache::cas("key", test2, cas_value); , memcache::key_not_stored );
-    BOOST_CHECK_NO_THROW ( mc << memcache::gets("key", retrieved, cas_value) );
+//    BOOST_CHECK_THROW    ( mc << memcache::cas("key", test2, cas_value); , memcache::key_not_stored );
+//    BOOST_CHECK_NO_THROW ( mc << memcache::gets("key", retrieved, cas_value) );
```

Comment:
I don't think I can accept this pull request which removes functionality from the tests and the library. Can you explain why you've removed "gets" support, as well as the "cas" feature?